riscv_ctg/generator.py

```diff
@@ -1083,7 +1083,16 @@ def __write_test__(self, file_name,node,label,instr_dict, op_node, usage_str):
             code.append("RVTEST_FP_ENABLE()")
         n = 0
         opcode = instr_dict[0]['inst']
-        op_node_isa = (op_node['isa']).replace('I','E',1) if 'e' in base_isa else op_node['isa']
+        op_node_isa = ""
+        extension = ""
+        if isinstance(op_node['isa'], list): 
```

Comment:
With all the nodes in the `template.yaml` having the same format, I think this particular `if-else` can be removed from the code. Only the code under the `if` portion is needed.

Reply:
The code also needs to account for replacing `I` with `E` incase the base ISA is `RV32E`.